src/project-state.py

```diff
@@ -266,11 +269,20 @@ def send_slack(project, text, attachment=None, color="#D3D3D3"):  # grey-ish
             "text": text,
             "footer": footer,
         }
-    response = slack.chat_postMessage(
-        channel=channel, attachments=[attachment]
-    )
-    print("...sent to channel %s" % channel)
-    return response
+
+    if use_slack_api:
```

Comment:
Since neither input is required, I think it might be nice to just add in a check if neither field is defined, or both is and fail with an appropriate message. By looking at the code, it looks like it will use the `postMessage` but not the webhook if both are defined.

Reply:
Great idea. Added this check above.